# Enumerated types
uint8 t_resetStatistics=0
uint8 t_start=1
uint8 t_pause=2
uint8 t_getResults=3
uint8 t_getDescriptors=4
uint8 t_enableLogging=5
uint8 t_disableLogging=6

uint8 givenCommand
---
float32[] results
string[] descriptors

